__tests__/app.test.js

```diff
@@ -106,4 +106,55 @@ describe("Nc News, testing API's", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    it("should increment the correct article's votes ", () => {
+      const newVote = 1;
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ incVotes: newVote })
+        .expect(201)
+        .then((res) => {
+          expect(res.body.article.votes).toBe(101);
```

Comment:
I would recommend you expand this test out and check the contents of the entire article which is returned, such as article_id, name, body and so on... and in the process also expecting votes to be 101 like it already says. This will give you additional confidence that it has both fetched the article, patched it, and returns the whole article intact apart from the votes increased.